web/src/vdom.ts

```diff
@@ -2,7 +2,43 @@ import _ from "lodash";
 
 import * as blueslip from "./blueslip";
 
-export function eq_array(a, b, eq) {
+// export type
+export type VDomNode = {
+    key: string;
+    eq: (other: VDomNode) => boolean;
+    render: () => string;
+    conversation: Conversation;
+};
+
+export type Attribute = [string, string][];
```

Comment:
This is not the type of an attribute, it’s the type of multiple attributes, so it should not be named `Attribute`.

Reply:
Thanks for the review, working on it soon